main/auth/justification.php

```diff
@@ -177,12 +177,26 @@
         }
         $actions .= '&nbsp;'.Display::url(get_lang('Delete'), api_get_self().'?a=delete_justification&justification_id='.$userJustification['id'], ['class' => 'btn btn-danger']);
         $table->setCellContents($row, $col++, $actions);
+        $code = $justification['code'];
+        $htmlHeadXtra[] = '<script type="text/javascript" >$(function(){$("#file_'.$code.'_file label").css("color","green");});</script>';
         $row++;
     }
 
     $userJustificationList .= $justificationContent.$table->toHtml();
 }
 
+$htmlHeadXtra[] = '<script type="text/javascript" >
+$(function(){
+    $("#justification label").each(function(){
+        var colorG = $(this).css("color");
+        var lgtxt = $(this).text().replace(/ /g,"").length;
+        if (colorG!="green"&&colorG!="rgb(0, 128, 0)"&&lgtxt>3) {
+            alert(lgtxt);
```

Comment:
I think this line should be removed because it generates an alert and open a popup in the navigator with a code in it and it is completely useless and blocking the user and blocking the css final loading of the page when using Chamilo Boost.